packages/api/test/unit/utils/parseAcceptHeader.test.ts

```diff
@@ -0,0 +1,26 @@
+import {expect} from "chai";
+import {parseAcceptHeader} from "../../../src/utils/server/parseAcceptHeader.js";
+import {ResponseFormat} from "../../../src/interfaces.js";
+
+describe("utils / parseAcceptHeader", () => {
+  describe("parseAcceptHeader", () => {
+    const testCases: {header: string | undefined; expected: ResponseFormat}[] = [
+      {header: undefined, expected: "json"},
+      {header: "application/json", expected: "json"},
+      {header: "application/octet-stream", expected: "ssz"},
+      {header: "application/invalid", expected: "json"},
+      {header: "application/octet-stream;q=0.5,application/json;q=1", expected: "json"},
+      {header: "application/octet-stream;q=1,application/json;q=0.1", expected: "ssz"},
+      {header: "application/octet-stream,application/json;q=0.1", expected: "ssz"},
+      {header: "application/octet-stream;,application/json;q=0.1", expected: "json"},
```

Comment:
Please also add a test case documenting the behavior when both `application/octet-stream` and `application/json` are both passed with `q=1`.  May also be helpful to show it with both orders to show that its order dependent and not preferential to one or the other

Reply:
Order dependence isn't actually part of the RFC, should I codify it with a test anyway?

Reply:
Yes please.  Unit tests are as much about documentation and also regression testing/notification as they are for meeting requirements.  I suppose both orders is not super critical but will highlight that the algorithm is order dependent and not preferential to one format or the other.